_episodes/21-carpentries.md

```diff
@@ -378,26 +378,26 @@ you are very welcome to put your name forward as a candidate.
 > What would you add, and what would you drop to make room?
 {: .challenge}
 
-[dc-blog]: {{ site.dc_site }}/blog/
+[dc-blog]: <{{ site.dc_site }}/blog/>
```

Comment:
Hm - `<url>` is Markdown's way of saying "show `url` but also make it a link".  The footer links (like this one) are a way of saying "I want to use a shortened form in the main text, and put the full URL at the bottom", so putting this in angle brackets shouldn't work.  I'll dig...